src/components/legend.ts

```diff
@@ -293,16 +297,15 @@ export module Components {
     }
 
     /**
-     * Gets the symbolFactoryAccessor of the legend, which dictates how
-     * the symbol in each entry is drawn.
+     * Gets the SymbolFactory accessor of the Legend, which determines the symbol for each entry.
```

Comment:
Maybe change to
"Gets the SymbolFactory accessor of the Legend.
The SymbolFactory accessor determines the symbol for each entry."